app/config/defaults.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
     packagist_host: ~
     packagist_metadata_dir: "%kernel.cache_dir%/composer-packages-build"
+    packagist_dumper_compress: 0
```

Comment:
ah sorry, I missed the fact that this was not just enabling the compression but also passed to ``gzencode`` itself